src/boot/bootctl.c

```diff
@@ -1345,18 +1361,53 @@ static int verb_list(int argc, char *argv[], void *userdata) {
                 log_info("No boot loader entries found.");
         else {
                 size_t n;
+                if (arg_json_format != JSON_FORMAT_OFF) {
+                        //Format flags
+                        BootEntry *entry = NULL;
+                        _cleanup_(json_variant_unrefp) JsonVariant *output;
+                        char *options = NULL;
+                        char *initrd = NULL;
+
+                        printf("{");
+                        if (arg_json_format == JSON_FORMAT_PRETTY)
+                                printf("\n");
+
+                        for (n = 0; n < config.n_entries; n++) {
+                                entry = config.entries + n;
+                                if (entry->options == NULL)
+                                        options = NULL;
+                                else
+                                        options = *(entry->options);
```

Comment:
this is not a signle string, but a NUL terminated array of strings (i.e. an "strv")